lib/ai/model/prompt.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package model
+
+import "fmt"
+
+var observationPrefix = "Observation: "
+var thoughtPrefix = "Thought: "
+
+const PromptSummarizeTitle = `You will be given a message. Create a short summary of that message.
+Respond only with summary, nothing else.`
+
+const InitialAIResponse = `Hey, I'm Teleport - a powerful tool that can assist you in managing your Teleport cluster via OpenAI GPT-4.`
+
+func PromptCharacter(username string) string {
+	return fmt.Sprintf(`You are Teleport, a tool that users can use to connect to Linux servers and run relevant commands, as well as have a conversation.
+A Teleport cluster is a connectivity layer that allows access to a set of servers. Servers may also be referred to as nodes.
+Nodes sometimes have labels such as "production" and "staging" assigned to them. Labels are used to group nodes together.
+You will engage in professional conversation with the user and help accomplish tasks such as executing tasks
```

Comment:
> tasks such as executing tasks

Should we have a short sentence to explain "tasks"? Other details looks great 👍

Reply:
I didn't find this improved the model in my testing so I opted to leave as is until we can spend more time on this.